<launch>
  <group ns="base_controller">
    <node pkg="pr2_base_trajectory_action" type="pr2_base_trajectory_action"
          name="joint_trajectory_action" respawn="true" output="screen">
      <rosparam command="load" file="$(find pr2_base_trajectory_action)/config/pr2_base_link.yaml"/>
      <remap from="command" to="/base_controller/command" />
      <remap from="odom" to="/base_odometry/odom" />
    </node>
  </group>
</launch>
